brainstormforce/ultimate-addons-for-gutenberg
# Fix: Social Share - Pinterest Share not working because missing image issue.

### Description

https://trello.com/c/UwgCz2d4/88-social-share-pinterest-share-not-working-because-missing-image

### Types of changes




Breaking change

### How has this been tested?

- Create Social share in master.
- Switch to this branch, add a featured image to the page check Pinterest share working.
- Create Reusable block use on a different pages and check its working.

### Checklist:
